megablocks/layers/mlp.py

```diff
@@ -501,11 +501,12 @@ def backward(ctx, ddsd_out):
         #
         # NOTE: This reuses the dactivation_fn_out allocation.
         if ctx.num_remat_bits == -1:
-            if activation_grad_fn is not None:
```

Comment:
I think there are a bunch more of these in this file to update like this?

Reply:
I thought I got them all in the last PR, but I'll check again.